feature-flags/feature-flags.development.ts

```diff
@@ -6,11 +6,8 @@ export const flagsDevelopment: FeatureFlagsInterface = {
   FEATURE_TRANSPARENCY_NEW_TABLE: true,
   FEATURE_CU_INDEX_NEW_TABLE: true,
```

Comment:
This one should be deleted as this is used to enable/disable the legacy cu overview table, which should be also deleted

Reply:
done